src/Libraries/RevitNodes/Elements/Views/View.cs

```diff
@@ -213,5 +211,33 @@ public Revit.Filter.OverrideGraphicSettings FilterOverrides(Revit.Filter.Paramet
         }
 
         #endregion
+
+        #region Scale
+
+        /// <summary>
+        /// Set View Scale
+        /// </summary>
+        /// <param name="scale">View scale is the ration of true model size to paper size.</param>
+        /// <returns name="View">View</returns>
+        public Element SetScale(int scale=100)
```

Comment:
I see a bunch of stuff coming in that I've looked at in another PR. I'll assume this is because one branch is a branch from the other. Unless these are different changes in some way?